4-projects/modules/base_env/remote.tf

```
@@ -41,6 +41,15 @@ data "terraform_remote_state" "bootstrap" {
   }
 }
 
+data "terraform_remote_state" "org_env" {
```

```suggestion
data "terraform_remote_state" "org" {
```
name needs to be fixed